projects/stef.md

```diff
@@ -0,0 +1,332 @@
+| batch_size: 128          | 1.910ms/msg    | 0.733ms/msg (x  2.61)   | 2.829ms/msg (x  0.68)    |
+| batch_size: 1024         | 14.524ms/msg   | 4.843ms/msg (x  3.00)   | 22.091ms/msg (x  0.66)   |
+| batch_size: 2048         | 27.970ms/msg   | 9.528ms/msg (x  2.94)   | 45.824ms/msg (x  0.61)   |
+| batch_size: 4096         | 44.902ms/msg   | 16.188ms/msg (x  2.77)  | 79.911ms/msg (x  0.56)   |
+| batch_size: 16384        | 137.348ms/msg  | 46.941ms/msg (x  2.93)  | 316.788ms/msg (x  0.43)  |
+| ======================== |                |                         |                          |
+| End-to-end               |                |                         |                          |
+| Total                    |                |                         |                          |
+| batch_size: 128          | 3.305ms/msg    | 2.961ms/msg (x  1.12)   | 9.369ms/msg (x  0.35)    |
+| batch_size: 1024         | 24.470ms/msg   | 17.612ms/msg (x  1.39)  | 75.186ms/msg (x  0.33)   |
+| batch_size: 2048         | 46.408ms/msg   | 34.148ms/msg (x  1.36)  | 176.430ms/msg (x  0.26)  |
+| batch_size: 4096         | 76.289ms/msg   | 57.735ms/msg (x  1.32)  | 397.859ms/msg (x  0.19)  |
+| batch_size: 16384        | 239.826ms/msg  | 165.299ms/msg (x  1.45) | 2291.036ms/msg (x  0.10) |
+
+To summarize, on these datasets Otel/STEF is the fastest when receiving and is the second 
+fastest when exporting.
+
+## Deliverables
+
+- stef-go: STEF supporting libraries for Go.
+- stef-gogrpc: STEF/gRPC protocol implementation in Go.
+- stef-pdata: Collector pdata <-> Otel/STEF converters.
+- Otel/STEF exporter and receiver, in
+  [open-telemetry/opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
+  repository.
+- stef-spec:  Otel/STEF format and protocol specification repository, with publication on
+  [https://opentelemetry.io/docs/](https://opentelemetry.io/docs/).
+
+## Staffing / Help Wanted
+
+The project already has all necessary staffing, see the next section. Additional 
+participants are welcome.
+
+### Required staffing
+
+- Project Lead, TC Sponsor, Go implementation engineer:
+[Tigran Najaryan](https://github.com/tigrannajaryan)
+- Second TC Sponsor: [Josh Suereth](https://github.com/jsuereth)
+- GC liaison: [Morgan McLean](https://github.com/mtwo)
+- Collector Maintainer: [Dmitrii Anoshin](https://github.com/dmitryax)
+
+## Timeline
+
+- 2025 Q2: Otel/STEF production grade implementation in Go, Otel Collector exporter and 
+  receiver in Development. Publish STEF specification in Development.
+- 2025 Q2: Add support for traces and logs.
+- 2025 Q3: Stabilization and improvements period.
+- 2025 Q4: Beta specification and implementation.
+- 2026 Q1+: Stable specification and implementation.
+
+## Project Board
+
+Project board link to be added here.
+
+## SIG Meetings and Other Info
+
+SIG meeting to be scheduled once the project is approved.
+
+## FAQ
```

Comment:
Has there been a comparative analysis done of other existing protocols besides Arrow?

Reply:
Benchmarks include comparison with OTLP, Parquet and Otel Arrow. If you think there are other interesting formats to compare to we can add it to the project goals.